### ISSUE TITLE
circle.yml: Change references of python3 to python

### ISSUE THREAD

**Issue by satwikkansal** [coala/coala-quickstart#106]
Use python3 instead of python similar to other coala repositories.

**Comment by jayvdb** [comment]
This one should wait until after #112 , and after Travis and Appveyor are testing lots of versions of Python, you could run the tests on only one version of Python on Circle CI, which means deps.sh can be merged into circle.yml.

**Comment by jayvdb** [comment]
What is the benefit of this?

The CI should all be in a venv , so `python` points to the correct version.

> Use python3 instead of python similar to other coala repositories and

coala & coala-bears appear to use python3 or pip3 in the documentation, but not very much in the CI, except as shebangs, and `.ci/tests.sh` which was removed from core and quickstart repos but is still in bears (I have a WIP to remove that file)

Based on the contents of https://github.com/coala/coala-quickstart/pull/234/files , if standardisation is the goal, mobanising travis yml would be the way to do that.
